config/routes.rb

```diff
@@ -2,5 +2,6 @@
   resources :spaces
   devise_for :users
   root to: 'pages#home'
+  resources  :storage
```

Comment:
not sure what this would be used for (seems to be a duplicate of : spaces)

Reply:
forgot to delete that 
Will do